src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/LastErrorTests.cs

```diff
@@ -28,7 +28,7 @@ public void SetLastPInvokeError_GetLastWin32Error()
         {
             int errorExpected = 123;
             Marshal.SetLastPInvokeError(errorExpected);
-            Assert.Equal(errorExpected, Marshal.GetLastWin32Error());
+            Assert.Equal(errorExpected, Marshal.GetLastPInvokeError());
```

Comment:
This change should be reverted. (Otherwise, the test would be redundant with `LastPInvokeError_RoundTrip`.)